### DISCUSSION TITLE
SOLD - All my SageTV gear (extenders+) - pay shipping costs

### DISCUSSION THREAD

**Discussion by tomahawk51** [OpenSageTV/Community#34] [Marketplace]
Hi all, I am looking to find a new home for the pile of various SageTV hardware I've amassed. What memories!

Please just message me if interested and an offer. See pics for details. 

![image](https://github.com/OpenSageTV/Community/assets/17829296/91c0ddce-31a7-4554-98d1-fb6ff07d14c1)
![image](https://github.com/OpenSageTV/Community/assets/17829296/b8fe94c6-c220-4b72-ae7d-b0e3204a8ddc)
![image](https://github.com/OpenSageTV/Community/assets/17829296/47a44986-16e5-4d27-b4d6-f789e1a747b8)
![image](https://github.com/OpenSageTV/Community/assets/17829296/b8ffdf06-3206-4461-9369-13d746cc4037)

**Comment by Sagetvmaster** [comment]
No takers yet?

**Reply by tomahawk51** [reply]
Nope! Still sitting in my basement. 

I'm new to GitHub, but assume people can direct message me if interested - right? (Update: works fine :) )

**Reply by Sagetvmaster** [reply]
I replied just so you'd know it works :)

**Reply by tomahawk51** [reply]
Hi all, any interest for purely the cost of shipping?

**Reply by eelpout** [reply]
yeah, I'd take an HD300. If there is direct messaging here, I can't find it.

**Reply by brookbphx** [reply]
Hi Tomahawk.   While I certainly don't need all of what you've shown, I'd be happy to take it off your hands for shipping cost and a bit extra for your trouble and to say thank you.  Any idea what shipping cost to Arizona would be?  

Or if you want to sell it piecemeal, you can see that lots of folks above are interested in particular pieces.  If you sell these individually, I'd be interested in a couple of HD300s and the HDPVR.

I think Github has removed the direct messaging by email capability but they do allow you to make your email address public.  That is likely not a good idea, as you'd likely start getting spammed from it if you made it public.  If you want to provide a way for people to contact you privately, you probably need to update your Github profile to include a website or social media link, and then people can contact you that way.  Or just check in on these comments to your discussion thread periodically.

**Reply by wnjj** [reply]
Yeah, not having PM really sucks for this particular kind of Open Source community sharing.  GitHub's explanation for removing PM is that information sharing & collaboration should be "open".  One option is to post an email here (ideally from the seller so multiple buyers can contact), then later edit the post to remove it once you made contact to minimize it getting grabbed.  Another is to use a disposable email that only check until the transaction is done.  Obviously anyone who reads the message before it's edited out could get access.

If want to use a normal email you can include spaces and spell parts of it out and use underscores that you can instruct the other person to remove like at "my_user_name at domain dot com (remove underscores)".  Basically an overly simple encryption that will only trip up non-humans.

**Reply by tomahawk51** [reply]
hi brookbphx - I'm inclined to sell all of it together, so responding to you.  I estimated shipping at ~$125 for East Coast US to CA per UPS/USPS.  So, maybe it would be a bit less? 

If interested, send me an email to vampirecastlevaniastuff at protonmail dot com

**Comment by tomahawk51** [comment]
> Hi Tomahawk. While I certainly don't need all of what you've shown, I'd be happy to take it off your hands for shipping cost and a bit extra for your trouble and to say thank you. Any idea what shipping cost to Arizona would be?
> 
> Or if you want to sell it piecemeal, you can see that lots of folks above are interested in particular pieces. If you sell these individually, I'd be interested in a couple of HD300s and the HDPVR.

Hi brookbphx,
I would like to see it all go together, as it's all packed up in a box now.  I estimated ~$130 shipping from the East Coast to CA, so I'd assume it's on that order.  If interested, please just send me an email 
and I can run a new estimate, etc.

To all, sorry for the delay - didn't check this post as soon as I should have!

**Reply by tomahawk51** [reply]
Sold.

**Comment by Sagetvmaster** [comment]
Excellent  news.  Glad they found a home :)


-------- Original message --------
From: tomahawk51 ***@***.***>
Date: 6/23/23 3:02 PM (GMT-08:00)
To: OpenSageTV/Community ***@***.***>
Cc: Sagetvmaster ***@***.***>, Comment ***@***.***>
Subject: Re: [OpenSageTV/Community] All my SageTV gear (extenders+) - pay shipping costs (Discussion #34)


Sold.
